packages/abi-gen-templates/partials/callAsync.handlebars

```diff
@@ -24,6 +24,10 @@ async callAsync(
         },
         self._web3Wrapper.getContractDefaults(),
     );
+    if (callDataWithDefaults.from !== undefined) {
+        callDataWithDefaults.from = callDataWithDefaults.from.toLowerCase();
+    }
+
```

Comment:
Yup, i could switch this to a ternary statement

Reply:
`prettier` is transforming these into 3-liners anyway because of line length though.

```
callDataWithDefaults.from = callDataWithDefaults.from
                ? callDataWithDefaults.from.toLowerCase()
                : callDataWithDefaults.from;
```

your snippet won't work because it says: if `callDataWithDefaults.from` is defined, use it as is. Otherwise, call `toLowerCase()` on it (when it's either `undefined` or empty string). 

Calling `toLowerCase()` on `undefined` will throw an error.